code/controllers/subsystem/chat.dm

```diff
@@ -0,0 +1,67 @@
+SUBSYSTEM_DEF(chat)
+	name = "Chat"
+	flags = SS_TICKER|SS_NO_INIT
+	wait = 1
+	priority = FIRE_PRIORITY_CHAT
+
```

Comment:
You also definitely want it to init for reasons mentioned here https://github.com/tgstation/TerraGov-Marine-Corps/pull/2134 , if you want to be safe you should be able to just copy the lastest version of that file from our  codebase.

Reply:
added